optaplanner-quickstarts/quarkus-school-timetabling/pom.xml

```diff
@@ -102,6 +102,11 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-test</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Looks like following the quarkus philosophy, we'll need to create a quarkus-optaplanner-test module...

Reply:
Sounds like a lot of hassle for something so small and barely significant. It's just a couple of classes.

Reply:
Yes. Same reasoning applies to quarkus-optaplanner-jackson etc though...